### ISSUE TITLE
Ability to save content

### ISSUE THREAD

**Issue by Silent-Hunter** [snoe/deedum#69]
I attempted to save an image so I could share it with my mother, who doesn't use Gemini (yet). But long tapping on the image does nothing, and there are no options for it anywhere else.

**Comment by snoe** [comment]
Yes, this would be a good addition. Other binary files (like mp3s) should also be downloadable.
